src/api/fetchData.js

```diff
@@ -0,0 +1,13 @@
+const home = '/data';
+async function fetchData(endpoint) {
+  const url = `${home}/${endpoint}`;
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    return console.log(error);
```

Comment:
why are you returning console.log() ???
It's return type is undefined (doesn't returns anything)